### ISSUE TITLE
Possible license incompatibility (Nordic and GPL)

### ISSUE THREAD

**Issue by fgaz** [meshtastic/firmware#4857]
This project contains both code under GPL 3.0 and code under Nordic 5-Clause.

GPL 3.0 states:

https://github.com/meshtastic/firmware/blob/10869ea10a8954d2471ae120aa0e4c76327c4fe7/LICENSE#L210-L212

https://github.com/meshtastic/firmware/blob/10869ea10a8954d2471ae120aa0e4c76327c4fe7/LICENSE#L222-L228

and

Nordic 5-Clause states:

https://github.com/meshtastic/firmware/blob/10869ea10a8954d2471ae120aa0e4c76327c4fe7/src/platform/nrf52/softdevice/ble.h#L21-L25

**Comment by jp-bennett** [comment]
Good catch. I know at least some other projects don't consider this a problem, but we'll look into it.
